worker/deployer/deployer.go

```diff
@@ -29,7 +29,7 @@ var _ logger = struct{}{}
 // to changes in a set of state units; and for the final removal of its agents'
 // units from state when they are no longer needed.
 type Deployer struct {
-	st       API
+	api      API
```

Comment:
some places we call this client some places we call it api. Would be good to be consistent for future changes.

Reply:
renamed